Sources/Component/Action/ActionView.swift

```diff
@@ -57,9 +64,37 @@ public final class ActionView<ContentView: StatefulViewProtocol>: StatefulView<A
     @objc
     private func updateBackgroundColor() {
         if [.highlighted, .selected].contains(button.state) {
-            button.backgroundColor = UIColor.black.withAlphaComponent(0.15)
+            switch model.highlightAnimation {
+            case .normal:
+                button.backgroundColor = UIColor.black.withAlphaComponent(0.15)
+
+            case .curveEaseInOut:
```

Comment:
I understand your solution but is there maybe a more generic solution? For example it could be possible that the animation block is passed to the view or a closure is executed with the current state or something similar. Then we could configure the highlight state to our special needs per app.

Reply:
This might be also useful but I would recommend having a set of already configured animations which can be extended throughout the lifetime of the view. For now I would leave it as is and add another feature request if needed.

Reply:
Why not adding the possibility directly? (If you have time 😉)

Reply:
Done.